pkg/getty/README.md

```diff
@@ -0,0 +1,64 @@
+# LinuxKit getty
+Image to create a getty on each console for a [linuxkit](https://github.com/linuxkit/linuxkit)-generated image.
+
+
+## Usage
+LinuxKit images do not create `getty` by default. If you want to be able to access a shell, you need to run a getty.
+
+Normally, unless you are running explicitly in a desktop version, LinuxKit images do not have swap enabled. If you want a console getty, add the following to your `moby.yml`:
+
+```
+services:
+  - name: getty
+    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
+```
+
+The above will launch a getty for each console defined in the cmdline, i.e. `/proc/cmdline`.
+
+
+### Login Options
+There are 3 ways to launch a getty on a linuxkit instance:
+
+1. Login disabled
+2. Password login
+3. Open access
+
+
+#### Login Disabled
+Login disabled prevents any console login. This is the most secure option and recommended for production deployments.
+
+To disable login entirely:
+
+1. Ensure you are running a version of `linuxkit/init` that has getty disabled.
```

Comment:
A little paranoid but a good thing. Since there are versions with getty, someone relying on it to not exist for security reasons should explicitly check.